content/docs/components/jupyter.md

```diff
@@ -4,6 +4,18 @@ description = "Using Jupyter notebooks in Kubeflow"
 weight = 10
 +++
 
+## Creating a custom Jupyter image
+You can create your own Jupyter image and use it in your Kubeflow cluster.
+Your custom image needs to meet the requirements created by Kubeflow Notebook Controller. Kubeflow Notebook Controlle  manges the life-cycle of notebooks. 
```

Comment:
Typo "Controller". Please check all the text for further typos, and let us know when you're ready for final review.

Reply:
done.